pkg/api/message/v1/service.go

```diff
@@ -366,13 +377,35 @@ func (s *Service) BatchQuery(ctx context.Context, req *proto.BatchQueryRequest)
 		logFunc = log.Info
 	}
 	logFunc("large batch query", zap.Int("num_queries", len(req.Requests)))
-	// NOTE: in our implementation, we implicitly limit batch size to 50 requests
-	if len(req.Requests) > 50 {
-		return nil, status.Errorf(codes.InvalidArgument, "cannot exceed 50 requests in single batch")
+
+	// NOTE: in our implementation, we implicitly limit batch size to 50 requests (maxSubscribeRequestLimitPerBatch = 50)
+	if len(req.Requests) > maxSubscribeRequestLimitPerBatch {
```

Comment:
`BatchQuery` and subscribe are unrelated endpoints. I'd suggest we give each their own constants/limits.

Reply:
I believe that @snormore asked for this variable name, although I'm very much open to renames. Do you want to suggest a different name ?

Reply:
Maybe `maxQueriesPerBatch`

Reply:
ok, done.